<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>251</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    border-style: none;
    border-radius: 5px;
    padding: 3px;
    background: transparent;
	border: 1px solid #3498db;
}
QTextEdit:focus {
    border: 2px solid #3498db;
}
QTextEdit:!enabled {
    color: #525252; 
} </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    border-style: none;
    border-radius: 5px;
    padding: 3px;
    background: transparent;
	border: 2px solid #3498db;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);
	border-style: solid;
	border-radius: 5px;	
	background-color: rgb(6, 163, 220);
}

QPushButton:hover{
	background-color: rgb(16, 196, 245);
}

QPushButton:pressed {
	background-color: rgb(3, 130, 176);
}</string>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clipboard_button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);
	border-style: solid;
	border-radius: 5px;	
	background-color: rgb(6, 163, 220);
}

QPushButton:hover{
	background-color: rgb(16, 196, 245);
}

QPushButton:pressed {
	background-color: rgb(3, 130, 176);
}</string>
   </property>
   <property name="text">
    <string>复制到粘贴板</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
      border: 2px solid grey;
      border-radius: 5px;
  }

  QProgressBar::chunk {
      background-color: #05B8CC;
      width: 20px;
  }</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	
	color: rgb(255, 255, 255);
	border-style: solid;
	border-radius: 5px;	
	background-color: rgb(6, 163, 220);
}

QPushButton:hover{
	background-color: rgb(16, 196, 245);
}

QPushButton:pressed {
	background-color: rgb(3, 130, 176);
}</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
